Newsweek Clone Homepage

edieatha opened the pull request:
## Newsweek home-page clone project: Building with Bootstrap.

1. Adde Bootstrap boilerplate code 
2. Used Bootstrap classes where possible.
3. Applied HTML semantics and CSS standard practices.
4. Responsive layout, so it won't brake on smaller screens.

## Built With

- HTML,
- Bootstrap
- CSS,
- Git

edieatha committed:
Created index.html and style.css

edieatha committed:
Create stylelint.config.js file

edieatha committed:
Updated stickle.yml file

edieatha committed:
Updated stylelint.config.js

edieatha committed:
Add bootstrap boilerplate code.

edieatha committed:
Updated stickler.yml file

edieatha committed:
Updated stickler.yml file

edieatha committed:
1. Added header and navbar section
2. Styling of header and navbar section.

edieatha committed:
Create header and navbar

edieatha committed:
Updated images folder

edieatha committed:
Upload images

edieatha committed:
Updated code for html

edieatha committed:
Delete old index.html

edieatha committed:
Added updated index.html

edieatha committed:
Updated css file

edieatha committed:
Uploaded favicon

edieatha committed:
1. Added social and search icons.
2. Additional styling

edieatha committed:
Updated Readme file

edieatha committed:
Added screenshot

edieatha committed:
Stickler linter errors fixed

edieatha committed:
HTML Validator checks done

trekab commented:
# Changes Required:
Hi @edieatha, great with the project so far! :+1: However, there are some changes you need to add to your project. These include;
- [x] [Your clone](https://rawcdn.githack.com/edieatha/newsweek/593339b3fb2aa8cb23721361b8477436cac05cb9/index.html) has two scroll bars. Get rid of the second scroll bar.
- On small screens, [your clone](https://rawcdn.githack.com/edieatha/newsweek/593339b3fb2aa8cb23721361b8477436cac05cb9/index.html)'s navigation appears to have smaller __sign in and subscribe buttons compared to the __Newsweek logo__. They should be the same size like on the [original site](https://www.newsweek.com/)
- [x] In the main section __TOP STORY__ should be on top on small screens. [Your clone](https://rawcdn.githack.com/edieatha/newsweek/593339b3fb2aa8cb23721361b8477436cac05cb9/index.html) has __FEATURED STORIES__.

__Kindly make the above changes and submit another code review request__
_Feel free to contact me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

edieatha committed:
Fixed small font-size for SIGNIN small screens.
2. Fixed Top story appearing on top in small screens.
3. Fixed horizontal scrollbar issue

edieatha committed:
Stickler error fixed

edieatha committed:
Stickler error fixed

edieatha commented:
## TSE Recommendations Complete:
1. Fixed 2 scrollbars on page
2. Fixed small "SIGN IN" on small displays
3. Fixed main section "TOP STORY" to come before "FEATURE STORIES" on small displays

edieatha committed:
Removed page padding

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

mikenath223 reviewed (approved):
### Hi @edieatha 👋

The layout of this project looks great.🚀

- [ ] OPTIONAL - Consider fixing the double left scrollbar that appears on the page as it seems to affect scrolling. [Screenshot]👇
![ice_screenshot_20200506-084524](https://user-images.githubusercontent.com/33323077/81149644-80521f80-8f76-11ea-8749-f23a02f982a2.png)

### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

edieatha committed:
Removed vertical scrollbar
